src/components/ui/FrogNft.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useEffect, useState } from "react";
+
+import { GRID_WIDTH_PX } from "features/game/lib/constants";
+
+import { frogAudio } from "lib/utils/sfx";
+import { loadFrogs } from "features/community/merchant/actions/loadFrogs";
+
+export const FrogNft: React.FC = () => {
+  const [showModal, setShowModal] = useState(false);
+  const [frogData, setFrogData] = useState([{ pixel_image: "" }]);
+
+  const open = () => {
+    setShowModal(true);
```

Comment:
There is no modal to open?

Reply:
Not yet there isn't 👀 (there will be in my new update)

Reply:
On second thought, it doesn't need a modal so I've removed it 😅. Thanks!